src/Form/input-classifiers.js

```diff
@@ -101,21 +123,6 @@ const isCCForm = (form) => {
     return textMatches?.length > 1
 }
 
-/**
- * Get a CC subtype based on selectors and regexes
- * @type (el: HTMLInputElement, form: HTMLFormElement) => string|undefined
- */
-const getCCFieldSubtype = (el, form) =>
-    CC_MATCHERS_LIST.find((sel) => checkMatch(el, form, sel))?.type
-
-/**
- * Get an identities subtype based on selectors and regexes
- * @type (el: HTMLInputElement, form: HTMLFormElement) => string|undefined
- */
-const getIDFieldSubtype = (el, form) =>
-    ID_MATCHERS_LIST.find((sel) => checkMatch(el, form, sel))?.type
-// TODO: Is it worth checking all selectors first before moving to labels and stuff?
```

Comment:
I had anticipated the work of `getSubtypeFromMatchers` in this `TODO`. We also no longer need these two different functions because we can call `getSubtypeFromMatchers` directly with different matchers.